### PR TITLE
Install j2objc (a google tool) to help build iOS and Android faster

### PR DESCRIPTION
**Pull request by trankimhieu**
Google develop this tool to help developer can build apps faster by use same logic code

### CODE REVIEW

File: Library/Formula/j2objc.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  sha256 "ff3e1da035dbaa4fc19e59cb74e3420b252bc328abb3b0e46acfce95416fdb3a"
+  head "https://github.com/google/j2objc.git"
+
+  depends_on "maven" => :build
+  depends_on :java => "1.7+"
+
+  def install
+
+    # Call make dist to build the full distribution
+    system "make", "dist", "-j4"
```
Comment (mistydemeo) [inline Library/Formula/j2objc.rb]: Don't hardcode the job count - Homebrew sets `MAKEFLAGS` with the appropriate `-j` already.

**Comment by trankimhieu** [comment]
Hi @mistydemeo 
I already removed job count flags.
Check it again please. Thanks so much :+1:

**Comment by mike-spainhower** [comment]
@trankimhieu Why did you close this?

**Comment by DomT4** [comment]
FYI: Already [exists](https://github.com/Homebrew/homebrew-devel-only/blob/master/Formula/j2objc.rb) in ` devel-only `; Is still ` pre-release `.

**Pull request closed without merging**